Build Failed with Detox on latest version: 2 files found with path 'lib/arm64-v8a/libc++_shared.so' from inputs

### Description

Upgraded RN from 75.4 to 76.2
Upgraded to latest react-native-guesture-handler
Using detox for E2E testing
Normal Build succeeds not Test Build

> Task :react-native-gesture-handler:mergeDebugAndroidTestNativeLibs FAILED

FAILURE: Build failed with an exception.

* What went wrong:                                                                                                                                                                                                               
Execution failed for task ':react-native-gesture-handler:mergeDebugAndroidTestNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
   > 2 files found with path 'lib/arm64-v8a/libc++_shared.so' from inputs:
      - C:\myfolder\myapp\node_modules\react-native-gesture-handler\android\build\intermediates\library_jni\debug\copyDebugJniLibsProjectOnly\jni\arm64-v8a\libc++_shared.so
      - C:\Users\e042739\.gradle\caches\8.10.2\transforms\bdca6cfc6ee61d54914f1ae561d11867\transformed\react-android-0.76.2-debug\jni\arm64-v8a\libc++_shared.so
     If you are using jniLibs and CMake IMPORTED targets, see
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 31m 3s
372 actionable tasks: 327 executed, 45 up-to-date





### Steps to reproduce

cd android && gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug

### Snack or a link to a repository

-

### Gesture Handler version

2.21.2

### React Native version

0.76.2

### Platforms

Android

### JavaScript runtime

None

### Workflow

React Native (without Expo)

### Architecture

Fabric (New Architecture)

### Build type

Debug mode

### Device

Android emulator

### Device model

_No response_

### Acknowledgements

Yes

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -153,6 +153,7 @@ android {
         exclude "**/libreact_render*.so"
         exclude "**/libreactnative.so"
         exclude "**/libjsi.so"
+        exclude "**/libc++_shared.so"
     }
 
     sourceSets.main {

